src/VisualStudio/Xaml/Impl/Implementation/XamlProjectService.cs

```diff
@@ -232,8 +239,13 @@ private void OnDocumentMonikerChanged(uint docCookie, IVsHierarchy hierarchy, st
             }
         }
 
-        private void OnDocumentClosed(string filePath)
+        private void OnDocumentClosed(string? filePath)
         {
```

Comment:
Is anyone else calling this? If not then we might as well put it in the event handler itself.

Reply:
Will do.

---
In reply to: [606385139](https://github.com/dotnet/roslyn/pull/52369#discussion_r606385139) [](ancestors = 606385139)

Reply:
The method is actually called in the other part of this partial type.

Reply:
If you mean the one inside OnBeforeLastDocumentUnlock, we have removed the call in this PR too :)

---
In reply to: [606391355](https://github.com/dotnet/roslyn/pull/52369#discussion_r606391355) [](ancestors = 606391355)

Reply:
@LinglingTong Ah, can indeed be put in the event handler itself then